### PR TITLE
[ADD]Auditlog Unlink Record

### PR DESCRIPTION
**Pull request by Hardik-OSI**

### CODE REVIEW

**Comment by bodedra** [comment]
Issue ref: https://github.com/OCA/server-tools/issues/2110

**Review by bodedra — APPROVED** [review]
Functional review. Worked as expected. Good work @Hardik-OSI 

With Capture Record = True

Screenshot 2021-06-21 at 09-53-50 Invoice - Odoo


With Capture Record = False

Screenshot 2021-06-21 at 09-57-35 Invoice - Odoo

File: auditlog/models/rule.py [hunk 1]
```diff
@@ -105,6 +105,8 @@ class AuditlogRule(models.Model):
     action_id = fields.Many2one(
         'ir.actions.act_window', string="Action",
         states={'subscribed': [('readonly', True)]})
+    capture_record = fields.Boolean("Capture Record",
+        help=("Select this if you want to keep track of Unlink Record"),)
```
Comment (kittiu) [inline auditlog/models/rule.py]: Not a blocker. Just make it common.
```suggestion
    capture_record = fields.Boolean(
        string="Capture Record",
        help="Select this if you want to keep track of unlinked record",
    )
```

**Comment by kittiu** [comment]
Nice feature :+1: 

Can you look into red travis, and add some test?

**Comment by bodedra** [comment]
@kittiu Thanks for your feedback. I have added test case and fixed red travis.

**Comment by kittiu** [comment]
@Saran440 you might like this PR.

**Review by kittiu — APPROVED** [review]

**Review by Saran440 — APPROVED** [review]
LGTM :+1:

**Comment by bodedra** [comment]
@OCA/tools-maintainers Any feedback?

**Review by sebalix — APPROVED** [review]

**Comment by kittiu** [comment]
> /ocabot merge nobump

Hi @dreispt just my curosity, this is a big improvement which includes a new database colum. Why this is `nobump`?

**Comment by dreispt** [comment]
@kittiu You're right, I was misled by the [ADD], that should have been [IMP].
The module version should have been bumped.

**Pull request merged**